src/webdriver_utils.py

```
@@ -1,212 +1,171 @@
-    manager = get_webdriver_manager()
-    with manager.driver_scope() as driver:
-        yield driver
\ No newline at end of file
+"""
+競馬データ収集ツール (Seleniumベース)
+
+netkeiba.comから指定された期間のレース情報を取得し、データベースに保存するツール。
+動的なページに対応するため、Selenium WebDriverを使用します。
+"""
+
+import argparse
```

Ruff (0.11.9)

`argparse` imported but unused

Remove unused import: `argparse`

(F401)